src/destination/destinationHandlers/sesHandler.ts

```diff
@@ -60,60 +104,53 @@ export class SESService implements Handler {
-            let sdk: NotifmeSdk = new NotifmeSdk({
-                channels: {
-                    email: {
-                        providers: [{
-                            type: 'ses',
-                            region: config['region'],
-                            accessKeyId: config['access_key'],
-                            secretAccessKey: config['secret_access_key'],
-                            //sessionToken: config['session_token'] // optional
-                        }]
-                    }
-                }
-            });
+    }
 
-            event.payload['fromEmail'] = config['from_email']
-            let engine = new Engine();
-            // let options = { allowUndefinedFacts: true }
-            let conditions: string = p['rule']['conditions'];
-            if (conditions) {
-                engine.addRule({conditions: conditions, event: event});
-                engine.run(event).then(e => {
-                    this.sendNotification(event, sdk, sesTemplate.template_payload).then(result => {
-                        this.saveNotificationEventSuccessLog(result, event, p, setting);
-                    }).catch((error) => {
-                        this.logger.error(error.message);
-                        this.saveNotificationEventFailureLog(event, p, setting);
-                    });
-                })
-            } else {
-                this.sendNotification(event, sdk, sesTemplate.template_payload).then(result => {
-                    this.saveNotificationEventSuccessLog(result, event, p, setting);
-                }).catch((error) => {
-                    this.logger.error(error.message);
-                    this.saveNotificationEventFailureLog(event, p, setting);
-                });
-            }
-        })
+    private async getDefaultConfig(){
+      try {
+        this.logger.info('getDefaultConfig')
+        const config = await this.sesConfigRepository.findDefaultSESConfig()
+        this.sesConfig = {
+          region: config['region'],
+          access_key: config['access_key'],
+          secret_access_key: config['secret_access_key'],
+          from_email: config['from_email']
+        }
+      } catch (error) {
+        this.logger.error('getDefaultConfig', error)
+        throw new Error('Unable to get default SES config');
+      }
     }
 
     public async sendNotification(event: Event, sdk: NotifmeSdk, template: string) {
         try {
+
+          this.logger.info('Notification send event payload')
+          this.logger.info(JSON.stringify(event.payload))
             let json = Mustache.render(template, event.payload)
-            json = JSON.parse(json)
             const res = await sdk.send(
                 {
-                    email: json
+                    email: JSON.parse(json)
                 }
             );
+
+            this.logger.info('Notification send')
+            this.logger.info(json)
```

Comment:
do we need these logs in prod

Reply:
For testing only, will remove before merging